src/release/breaking-changes/android-14-nonlinear-text-scaling-migration.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Android 14 nonlinear font scaling enabled after v3.14
+description: >
+  New Android 14 nonlinear font scaling feature is enabled in Flutter after v3.14.
+---
+
+## Summary
+
+Android 14 introduced nonlinear font scaling to 200%. It may change how your app 
+looks when the user changes the accessiblity text scaling in system preferences.
+
+## Background
+
+The [Android 14 nonlinear font scaling][] feature prevents excessive accessibility 
+font scaling by scaling larger text at a lesser rate when the user increases the 
+text scaling value in system preferences.
+
+## Migration guide
+
+As suggested by the 
+[Android 14 feature overview][Android 14 nonlinear font scaling], perform UI 
+testing with the maximum font size enabled (200%) to ensure that your app 
+applies the font sizes correctly and can accommodate larger font sizes without
+impacting usability.
+
+Consider migrating from `textScaleFactor` to `TextScaler` following this 
+migration guide: [Deprecate `textScaleFactor` in favor of `TextScaler`][], to
+adopt nonlinear font scaling in your app and custom widgets.
```

Comment:
**chore:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). 
Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
To adopt nonlinear font scaling in your app and custom widgets,
consider migrating from `textScaleFactor` to `TextScaler`.
This migration guide outlines the process:
[Deprecate `textScaleFactor` in favor of `TextScaler`][], .
```